trailblazer/server/api.py

```diff
@@ -231,24 +231,23 @@ def post_mark_analyses_deleted():
 
 @blueprint.route("/add-pending-analysis", methods=["POST"])
 def post_add_pending_analysis():
-    """Add new analysis with pending status"""
-    content = request.json
+    """Add new analysis with status: pending."""
     try:
-        analysis_obj = store.add_pending_analysis(
-            case_id=content.get("case_id"),
-            email=content.get("email"),
-            type=content.get("type"),
-            config_path=content.get("config_path"),
-            out_dir=content.get("out_dir"),
-            priority=content.get("priority"),
-            data_analysis=content.get("data_analysis"),
-            ticket_id=content.get("ticket"),
-            workflow_manager=content.get("workflow_manager"),
+        analysis: Analysis = store.add_pending_analysis(
+            case_id=request.json.get("case_id"),
+            email=request.json.get("email"),
+            type=request.json.get("type"),
+            config_path=request.json.get("config_path"),
+            out_dir=request.json.get("out_dir"),
+            priority=request.json.get("priority"),
+            data_analysis=request.json.get("data_analysis"),
+            ticket_id=request.json.get("ticket"),
+            workflow_manager=request.json.get("workflow_manager"),
```

Comment:
I do not like context since it does not provide any meaningful information except that something is ...well in "context". It does not say what the "context" is or where the data is coming from. How about ´post_request´ instead of context?